precice.pyx

```diff
@@ -124,12 +388,39 @@ cdef class Interface:
         vertex_id = self.thisptr.setMeshVertex(mesh_id, <const double*>_position.data)
         return vertex_id
 
-    # returns the number of vertices of a mesh
     def get_mesh_vertex_size (self, mesh_id):
+        """
+        Returns the number of vertices of a mesh
+
+        Parameters:
+            mesh_id (int): ID of the mesh
+
+        Returns:
+            sum (int): amount of vertices of the mesh
+        """
         return self.thisptr.getMeshVertexSize(mesh_id)
 
-    # creates multiple mesh vertices
     def set_mesh_vertices (self, mesh_id, positions):
+        """
+        Creates multiple mesh vertices
+
+        Parameters:
+            mesh_id (int): ID of the mesh to add the vertices to.
+            positions (pointer): a pointer to the coordinates of the vertices
+                                 The 2D-format is (d0x, d0y, d1x, d1y, ..., dnx, dny)
+                                 The 3D-format is (d0x, d0y, d0z, d1x, d1y, d1z, ..., dnx, dny, dnz)
```

Comment:
This has to be updated to the format that the python bindings are using. I.e a `numpy` array of shape `N x D`. Consider giving an example, as well. Compare to `help(np.array)`, where at the bottom of the documentation examples are provided.

Reply:
All the block commands and also the relevant `set` and `get` commands have the updated `numpy.array([N, D])` documented along with examples.